dashboard/app/cron.yaml

```diff
@@ -25,3 +25,6 @@ cron:
 # Update other coverage numbers every day.
 - url: /cron/batch_coverage?days=true&months=true&steps=10
   schedule: every 24 hours
+# Export reproducers every week.
+- url: /cron/batch_reproexport
```

Comment:
`batch_reproexport` is not mentioned anywhere else in the repository. Is there a way we could store all the batch configuration in git?

Reply:
batch_main.cpp:20: http.HandleFunc("/cron/batch_reproexport", handleBatchReproExport)

What do you mean it is not mentioned?

Reply:
Ah, right :)